plugins/org.komodo.eclipse.workspace.ui/src/org/komodo/eclipse/workspace/ui/common/util/StyledTextEditor.java

```diff
@@ -0,0 +1,1012 @@
+/*
+ * JBoss, Home of Professional Open Source.
```

Comment:
StyledTextEditor already exists in eclipse.sql.ui. Rather than adding a second, can we refactor them both into a common plugin?